src/main/java/com/greedobank/cards/controller/BankAccountController.java

```diff
@@ -0,0 +1,69 @@
+package com.greedobank.cards.controller;
+
+import com.greedobank.cards.dto.BankAccountCreationDTO;
+import com.greedobank.cards.dto.BankAccountDTO;
+import com.greedobank.cards.dto.BankAccountGetBalanceDTO;
+import com.greedobank.cards.dto.BankAccountReplenishDTO;
+import com.greedobank.cards.dto.BankAccountWithdrawDTO;
+import com.greedobank.cards.dto.BankAccountWithdrawOnlineDTO;
+import com.greedobank.cards.service.BankAccountService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = "/api/v1/account")
```

Comment:
when we change the version with some router (green-blue deployment or smth), we will have to change it everywhere?

Reply:
I think, when we change at least one endpoint - version changes